test/config/config-logging.yaml

```diff
@@ -0,0 +1,54 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-logging
```

Comment:
So for the main file, we symlink it under `testdata` and validate it's content in a Go test.  Can we do the same here to validate changes?  Otherwise this looks good to me

Reply:
@mattmoor Sounds good. Should it be a new test under `test/`? Which package should it belong?

Reply:
I'd just put it right next to where we test the real config (same test, second symlink)

Reply:
@mattmoor sounds good! Just added the test and symlink